board/freescale/imx8mm_evk/spl.c

```diff
@@ -291,24 +291,13 @@ unsigned long spl_mmc_get_uboot_raw_sector(struct mmc *mmc,
 	int boot_secondary = boot_mode_getprisec();
 	unsigned long offset = CONFIG_SYS_MMCSD_RAW_MODE_U_BOOT_SECTOR;
 
-	if (boot_secondary)
+	if (boot_secondary) {
 		offset += CONFIG_SECONDARY_BOOT_SECTOR_OFFSET;
-
-	debug("%s: Booting from 0x%lx offset as GPR10 "
-	      "PERSIST_SECONDARY_BOOT bit = %d\n",
-	      __func__, offset, boot_secondary);
-
-	if (!boot_secondary) {
-		/*
-		 * Set GPR10 PERSIST_SECONDARY_BOOT bit,
-		 * so in case of hang or other issues
-		 * BootROM will load recovery boot images
-		 * after _warm_ reset.
-		 */
-		boot_mode_enable_secondary(true);
-
-		debug("%s: Set: GPR10 PERSIST_SECONDARY_BOOT = %d\n",
-		      __func__, boot_mode_getprisec());
+		printf("SPL: Booting secondary boot path: using 0x%lx offset "
```

Comment:
Should we change these to `debug` ?

Reply:
To be honest I'd prefer to have that information always printed